docs/modules/ROOT/pages/server.adoc

```diff
@@ -17,6 +17,442 @@ include::includes/want-to-contribute.adoc[]
 
 include::./includes/server-getting-started.adoc[leveloffset=+1, opts=optional]
 
+[[codegen-extensions]]
+== Codegen Extensions
+
+Apicurio Codegen Extensions use https://swagger.io/docs/specification/v3_0/openapi-extensions/[OpenAPI specification extensions] to allow customization of the generated source code based on your API contract.
```

Comment:
I might prefer a non-swagger link here.  Maybe something like this:

https://spec.openapis.org/oas/v3.0.3.html#specification-extensions